functions/src/utils/venue.js

```diff
@@ -3,6 +3,9 @@ const { VenueIdSchema } = require("../types/venue");
 const VENUE_ID_REGEX = /[a-z0-9_]{1,250}/;
 const INVALID_SLUG_CHARS_REGEX = /[^a-zA-Z0-9]/g;
 
+/**
+ * @deprecated Use generateSlug instead.
+ */
```

Comment:
Please don't use 
```js
const generateSlug = (name) => {
  return name.replace(/\W/g, "").toLowerCase();
};
```

```suggestion
// @debt rename this one to generateSlug
```

`getVenueId()` has the correct implementation as staging:

![sparkle-generate-slug-01](https://user-images.githubusercontent.com/79229621/140527759-3fe8cd6a-3d78-411b-a4eb-7ef5978089bd.png)

Reply:
My bad, I'll fix it/

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2474/commits/cf4233edc137706c2ccb7d1ce357c2591cba2370